spec/facets/ProcessingTypeJobFacet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://openlineage.io/spec/facets/2-0-2/ProcessingTypeJobFacet.json",
+  "$defs": {
+    "ProcessingTypeJobFacet": {
+      "allOf": [
+        {
+          "$ref": "https://openlineage.io/spec/2-0-2/OpenLineage.json#/$defs/JobFacet"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "jobType": {
+              "description": "Job processing type.",
+              "type": "string",
+              "default": "BATCH",
+              "enum": ["BATCH", "STREAMING"]
```

Comment:
Can we do just `string` without enum here, and specify batch/streaming otherwise? Enums are cool but in schemas they introduce problems with forward compatibility.

Reply:
Aren't we able to extend enum in the future if we like?

Reply:
you're right. lets move this to strings.